web-wallet/src/lib/components/Transactions/Transactions.svelte

```diff
@@ -1,25 +1,37 @@
 <script>
-	import { mdiContain	} from "@mdi/js";
+	import { mdiArrowLeft, mdiContain } from "@mdi/js";
 	import { fade } from "svelte/transition";
 	import { logo } from "$lib/dusk/icons";
 	import {
-		Anchor, Badge, Icon, Tooltip
+		Anchor, AnchorButton, Badge, Card, Icon, Throbber, Tooltip
 	} from "$lib/dusk/components";
 	import { createFeeFormatter, createTransferFormatter } from "$lib/dusk/currency";
-	import { settingsStore } from "$lib/stores";
 	import { calculateAdaptiveCharCount, middleEllipsis } from "$lib/dusk/string";
 	import { onMount } from "svelte";
+	import { sortByHeightDesc } from "$lib/transactions";
+	import { compose, take } from "lamb";
```

Comment:
Can we adopt the convention of having, in order:
1. `node_modules` imports
2. `$lib` imports
3. "local" imports

I'd like also to have a blank newline between the blocks, but @kieranhall expressed doubts about it in the past.

cc @kieranhall @deuch13

Reply:
Similarly as my point above, I think it will become too many things for the developer to be keeping track of. Furthermore, the code editors suggest the imports and for the developer having to manually reorder them somewhat voids the benefits we are getting from the auto-import.

Reply:
I think having tidy code should be the primary thing to keep track of, but that's just my opinion: let's hear from the others as in these cases we need a general consensus.

Reply:
I'm all in for well-written and maintainable code, but I don't see benefits from the proposal, especially given the extra time that each developer and each reviewer will have to spend to police if the rule has been applied or not. To me, using imports in a different order does not equal to "untidy" code. There are things such as ensuring that imports from the same modules are combined, etc, which make sense. 

I think our time should be spent on solving actual issues, rather than becoming too overly focused on such details. But let's see what others have to say as well.

Reply:
> Can we adopt the convention of having, in order:
> 
> 1. `node_modules` imports
> 2. `$lib` imports
> 3. "local" imports
> 
> I'd like also to have a blank newline between the blocks, but @kieranhall expressed doubts about it in the past.
> 
> cc @kieranhall @deuch13

Personally, I prefer to not have blank lines around code of the same _purpose_.  If something is multi-line then I think a blank line can improve readability.  This is just a preference, though as there is no optimization gain from this.

What I would prefer more would be [linting rules that could be enforced](https://github.com/dusk-network/dusk-eslint-config/issues/12).   However, it remains to be seen if this would conflict with Prettier or not.

TLDR; I'm not too bothered and am, in any case, happy to group how you suggested above.